docker-compose.yml

```diff
@@ -6,26 +6,18 @@ services:
       context: .
       dockerfile: dev.Dockerfile
     env_file: .env.development
-    environment:
-      RUST_LOG: ${RUST_LOG}
     ports:
       - 7878:7878
     volumes:
-      - ${PWD}/src:/app/src
-      - ${PWD}/target:/app/target
-      - ${PWD}/Cargo.lock:/app/Cargo.lock
-      - ${PWD}/Cargo.toml:/app/Cargo.toml
+      - ${PWD}:/app
+      - ${PWD}/docker/volumes/target:/app/target
```

Comment:
Don't you want to map this directly to the `target/` you have in the PWD? like:

```
${PWD}/target:/app/target
```

Reply:
I can do that and I think that in general this is fine, but for some installations of docker. It can create those files with the root user and therefore it might create conflicts if you want to run the service locally.

Reply:
Bu service you mean running this crate in your machine with `cargo run` for instance?

Reply:
Yes, if we run cargo run, and in the files in target are from the root user, the process may fail due to permission issues.

Reply:
Im thinking about the `Cargo.*` files, I mean, it isn't intuitive to have `Cargo.*` files in the PWD but docker use the same `Cargo.lock` for instance and leaves the binaries on `PWD/docker/volumes/target` instead?

So I rather that being more straight forward and keep the same directory structure as much as possible to avoid duplicating things.

Another thing is that `target/` based on the crates you use can get very "nasty", and having that much files duplicated is not very optimized

Reply:
Ok, I removed that target volume. So, we are sharing now the target/ directory.